caleb-lab/controllers/person-controller.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
```

Comment:
**TODO FOR LOST POINTS**

This is a top-level comment that you need your gitignore, eslintrc, and a README. Also, take out comments and console.logs before submission. Put them in a separate branch for your own reference.